docs/source/integrations/lightning_flash.rst

```diff
@@ -509,48 +557,67 @@ Flash models with FiftyOne serializers will directly return predictions as
 
 .. note::
 
-    FiftyOne serializers have an optional
-    :class:`return_filepath <flash:flash.core.classification.FiftyOneLabels>`
-    flag that supports returning dicts that contain both the |Label| objects
-    and the ``filepath`` of the associated media.
+    FiftyOne outputs have an optional
+    :class:`return_filepath=False <flash:flash.core.classification.FiftyOneLabelsOutput>`
+    flag that supports returning a list of |Label| objects corresponding to the
+    sample ordering of the ``predict_dataset`` rather than the default dicts
+    that contain both the |Label| objects and the ``filepath`` of the
+    associated media.
 
 Specifying class names
 ----------------------
 
 Generally, Flash model checkpoints will contain the class label strings for the
 model. However, if necessary, you can also explicitly pass the labels to most
-:class:`Serializer <flash:flash.core.data.process.Serializer>` instances,
-FiftyOne-style serializers included:
+:class:`Output <flash:flash.core.data.io.output.Output>` instances,
+FiftyOne-style outputs included:
 
 .. code-block:: python
     :linenos:
 
     import fiftyone as fo
     import fiftyone.zoo as foz
 
-    from flash.image import ObjectDetector
-    from flash.image.detection.serialization import FiftyOneDetectionLabels
+    from flash import Trainer
+    from flash.image import ImageClassificationData, ImageClassifier
+    from flash.core.classification import FiftyOneLabelsOutput
 
     # Load your dataset
-    dataset = foz.load_zoo_dataset("quickstart", max_samples=5)
+    dataset = foz.load_zoo_dataset("quickstart", max_samples=5).clone()
```

Comment:
suggestion: remove `clone()` for simplicity?

I sometimes include it when using the entire `quickstart` dataset in an example that adds/edits fields, since many examples use the full dataset. But, when only a subset is being used, clashes are less likely